container-toolkit/supported-platforms.md

```diff
@@ -9,28 +9,44 @@
 
 ## Linux Distributions
 
-Supported Linux distributions are listed below:
-
-| OS Name / Version        | amd64 / x86_64 | ppc64le | arm64 / aarch64 |
-| ------------------------ | -------------- | ------- | --------------- |
-| Amazon Linux 2023        | X              |         | X {sup}`1`      |
-| Amazon Linux 2           | X              |         | X               |
-| Open Suse/SLES 15.x      | X              |         |                 |
-| Debian Linux 10          | X              |         |                 |
-| Debian Linux 11          | X              |         |                 |
-| Centos 7                 | X              | X       |                 |
-| Centos 8                 | X              | X       | X               |
-| RHEL 7.x                 | X              | X       |                 |
-| RHEL 8.x                 | X              | X       | X               |
-| RHEL 9.x                 | X              | X       | X               |
-| Ubuntu 18.04             | X              | X       | X               |
-| Ubuntu 20.04             | X              | X       | X               |
-| Ubuntu 22.04             | X              | X       | X               |
-| Ubuntu 24.04             | X              |         | X               |
-
-The `arm64` / `aarch64` architecture includes support for Tegra-based systems.
-
-1. For Amazon Linux 2023 on Arm64, a `g5g.2xlarge` Amazon EC2 instance was used for validation.
+Releases are qualification-tested and expected to work on these platforms:
+
+| OS Name / Version        | amd64 / x86_64 | ppc64le | arm64 / aarch64 {sup}`1` |
+| ------------------------ | -------------- | ------- | ------------------------ |
+| Amazon Linux 2023        | X              |         | X {sup}`2`               |
+| Amazon Linux 2           | X              |         | X                        |
+| Open Suse/SLES 15.x      | X              |         |                          |
+| Debian Linux 10          | X              |         |                          |
+| Debian Linux 11          | X              |         |                          |
+| CentOS 7                 | X              | X       |                          |
+| CentOS 8                 | X              | X       | X                        |
+| RHEL 7.x                 | X              | X       |                          |
+| RHEL 8.x                 | X              | X       | X                        |
+| RHEL 9.x                 | X              | X       | X                        |
+| Ubuntu 18.04             | X              | X       | X                        |
+| Ubuntu 20.04             | X              | X       | X                        |
+| Ubuntu 22.04             | X              | X       | X                        |
+| Ubuntu 24.04             | X              |         | X                        |
+
+
+Note:
```

Comment:
Maybe lets put this in its own section? or as some paragraphs after the table. Otherwise the footnotes get pushed down further and we have a bit of note inception going on, notes within notes.

Maybe something like "Using unsupported distros" or "Distribution testing" or "Additional distros" for a section heading. Then we can go into details on the qualification testing done and your note on using unsupported distros how to give us feedback?

Reply:
> Maybe lets put this in its own section? or as some paragraphs after the table.

Yes, I like that you say this. Thought about it too, was looking for feedback about this in particular. 

I will add sub sections, one about reporting issues in general, and one about "Other Linux distributions".